msal/src/main/AndroidManifest.xml

```diff
@@ -9,20 +9,20 @@
         <!-- MSAL activity that will be used to process all the auth related logic  -->
         <activity
             android:name="com.microsoft.identity.common.internal.providers.oauth2.AuthorizationActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout"
+            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout|keyboard"
             android:exported="false"
             android:launchMode="singleTask" />
 
         <activity
             android:name="com.microsoft.identity.common.internal.providers.oauth2.CurrentTaskAuthorizationActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout"
+            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout|keyboard"
             android:exported="false"
             android:launchMode="standard" />
 
         <!-- Helper activity for displaying current broker redirect URI configuration -->
         <activity
             android:name="com.microsoft.identity.client.helper.BrokerHelperActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout"
+            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout|keyboard"
```

Comment:
Why do we need to add this to BrokerHelperActivity? Adding is probably harmless but I'm curios if it is actually needed?

Reply:
I mean I think we are not discovering USB devices on this activity, right?

Reply:
I'll double check if the flag is needed for this activity

Reply:
Based on my testing, wasn't needed. Removed it.